### PR TITLE
version 0.14.0 and change log

### PR DESCRIPTION
**Pull request by nejisama**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,45 @@
...
+- Metrics for adding HTTP StatusCode [@dengqian]( https://github.com/dengqian)
+- Add Metrics Admin API output [@dengqian](https) ://github.com/dengqian)
+- New interface to query the number of current requests for proxy [@zonghaishang](https: //github.com/zonghaishang)
+- Support for HostRewrite Header [@liangyuanpeng]( https://github.com/liangyuanpeng)
+
+### Optimization
+
+- Upgrade tars dependencies to fix compilation issues with higher versions of Golang [@ wangfakang](https://github.com/wangfakang)
+- xDS Configuration Analysis Upgrade Adaptation Istio 1.5.x [@wangfakang]( https://github.com/wangfakang)
+- Optimize log output from proxy [@wenxuwan](https://github .com/wenxuwan)
```
Comment (rootsongjc) [inline CHANGELOG.md:27]: 链接了多了个空格

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+- DNS Cache default time changed to 15s [@wangfakang](https) ://github.com/wangfakang)
```
Comment (rootsongjc) [inline CHANGELOG.md:28]: 链接格式错误
